src/routes/api.js

```diff
@@ -238,16 +239,41 @@ function openSocket() {
     console.log(`Connected to car server: ${client.remoteAddress}:${CAR_PORT}`);
   });
 
-  // Data received listener
-  client.on("data", (data) => {
-    if(data.length === bytesPerPacket) {
-      //console.time("update data");
-      unpackData(data);
-      //console.timeEnd("update data");
-    } else {
-      console.warn("ERROR: Bad packet length ------------------------------------");
-    }
-  });
+ // Data received listener
+ let partialData = '';
+
+ client.on("data", (data) => {
+   // Combine the new data with any previously received partial data
+   partialData += data.toString('latin1');
+
+   let startIndex = partialData.indexOf("<bl>");
+   let endIndex = partialData.indexOf("</bl>");
+
+   while (startIndex !== -1 && endIndex !== -1) {

```

Comment:
Let's add a check here to see if `partialData.length === bytesPerPacket` (and maybe that `startIndex !== 0` and/or `endIndex !== bytesPerPacket + 4`?). If so, pass it on as is. This will make the API backwards compatible by allowing packets without \<bl\>\</bl\> to pass through to the chase car dashboards